bugzilla2gitlab/utils.py

```diff
@@ -41,7 +41,7 @@ def markdown_table_row(key, value):
     '''
     Create a row in a markdown table.
     '''
-    return "| {} | {} |\n".format(key, value)
+    return "| " + key + " | " + value + " |\n"
```

Comment:
Why was this changed? Please revert to use the "new" format style.

Reply:
When table row contains non-ascii letters this causes an exception (in my case I had an attachment with non-ascii description but it might be also localized date string etc.). Now I see that it would be enough to explicitly make formatting template a unicode string: 
`    return u"| {} | {} |\n".format(key, value)`
Pushed this variant, please check.